app/components/ThinkingMessage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Flex } from "@chakra-ui/react";
+
+export default function ThinkingMessage() {
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      bg="whiteAlpha.200"
+      color="fg"
+      px={4}
+      py={3}
+      borderRadius="lg"
+      maxW="80%"
+      mb={4}
```

Comment:
```suggestion
      mb={4}
```
I think the only styling needed here is the margin; it's not a button, and there is only one child.